.github/workflows/ci.yml

```diff
@@ -35,10 +35,10 @@ jobs:
         with:
           ocaml-version: ${{ matrix.ocaml-version }}
       - name: Install dependencies
-        if: steps.cache-opam.outputs.cache-hit != 'true'
```

Comment:
@mikonieminen @joseferben This line prevents alcotest from being installed, so I deleted it. Is there another way to circumvent that?

Reply:
I usually install in manually in a prior step.

Reply:
Where would you put it, do you have an example?

Reply:
Here we install some system deps before everything else: https://github.com/oxidizing/sihl/blob/master/.github/workflows/ci.yml#L43

And then we run `make deps` which install a bunch of dev dependencies. 

```
.PHONY: deps
deps:
	opam install -y dune-release merlin ocamlformat utop
	opam install -y alcotest-lwt mariadb.1.1.4 caqti-driver-postgresql.1.8.0 caqti-driver-mariadb.1.8.0
	opam install . -y --deps-only --locked
	eval $(opam env)
```

Reply:
Looking at the CI file, I think you can update to `uses: avsm/setup-ocaml@v2` and then omit those anyway.

Reply:
```
     - name: Use OCaml ${{ matrix.ocaml-version }}
        uses: ocaml/setup-ocaml@v2
        with:
          ocaml-compiler: ${{ matrix.ocaml-compiler }}
          dune-cache: true
```

Reply:
This does not seem to work, I think we still need to install dependencies somewhere?